packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -117,7 +117,119 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
   }
 `;
 
+const syntaxHighlighting = (
+  colors: FrontityOrg["state"]["theme"]["colors"]
+) => css`
+  /* This is a modified theme for Prism.js
+  
+  /* PrismJS 1.19.0
+      https://prismjs.com/download.html#themes=prism-dark&languages=markup+clike+javascript+jsx 
+  */
+
+  code[class*="language-"],
+  pre[class*="language-"] {
+    font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
+    font-size: 0.78rem;
+    text-align: left;
+    white-space: pre;
+    word-spacing: normal;
+    word-break: normal;
+    word-wrap: normal;
+    line-height: 1.65;
+    color: ${colors.wall};
+
+    -moz-tab-size: 4;
+    -o-tab-size: 4;
+    tab-size: 4;
+
+    -webkit-hyphens: none;
+    -moz-hyphens: none;
+    -ms-hyphens: none;
+    hyphens: none;
+  }
+
+  @media print {
+    code[class*="language-"],
+    pre[class*="language-"] {
+      text-shadow: none;
+    }
+  }
+
+  .token.comment,
+  .token.prolog,
+  .token.doctype,
+  .token.cdata {
+    color: hsl(30, 20%, 50%);
+  }
+
+  .token.punctuation {
+    opacity: 0.7;
+  }
+
+  .token.namespace {
+    opacity: 0.7;
+  }
+
+  .token.property,
+  .token.tag,
+  .token.boolean,
+  .token.number,
+  .token.constant,
+  .token.symbol {
+    color: ${colors.red};
+  }
+
+  .token.selector,
+  .token.attr-name,
+  .token.string,
+  .token.char,
+  .token.builtin,
+  .token.inserted {
+    color: ${colors.grass};
+  }
+
+  .token.operator,
+  .token.entity,
+  .token.url,
+  .language-css .token.string,
+  .style .token.string,
+  .token.variable {
+    color: ${colors.orange};
+  }
+
+  .token.atrule,
+  .token.attr-value,
+  .token.keyword {
+    color: ${colors.red};
```

Comment:
I would use here `colors.orange` until we confirm new colors with the designer. The design is using orange right now.

Reply:
actually, I'm already using `orange` here: 
```css
  .token.operator,
  .token.entity,
  .token.url,
  .language-css .token.string,
  .style .token.string,
  .token.variable {
    color: ${colors.orange};
  }
```

So, for now I ll keep it as is, but let's wait for the designer to update the colors 🙂 .